plinth/modules/diaspora/__init__.py

```diff
@@ -48,7 +48,7 @@ def get_configured_domain_name():
 
 version = 1
 
-title_en = 'Federated Social Network (diaspora*)'
+title_en = 'Federated Social Network (diaspora*)\n'
```

Comment:
@kvaghel1 The newline should go in-between the descriptive name and the module name. Please check how it is done for other modules. For example: https://github.com/freedombox/Plinth/blob/master/plinth/modules/mumble/__init__.py#L35